circle.yml

```diff
@@ -1,36 +1,55 @@
-machine:
-  java:
-    version: oraclejdk8
-  environment:
-    TERM: dumb
-  post:
-    # install JCE Unlimited Strength Jurisdiction Policy Files
-    - |
-      wget -qO- --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" "http://download.oracle.com/otn-pub/java/jce/8/jce_policy-8.zip" | bsdtar -xvf- -C /tmp && sudo cp /tmp/UnlimitedJCEPolicyJDK8/* $JAVA_HOME/jre/lib/security/
-
-general:
-  artifacts:
-    - encrypted-config-value/build/libs
-    - encrypted-config-value-bundle/build/libs
-    - encrypted-config-value-bundle-dropwizard1/build/libs
-
-dependencies:
-  override:
-    - ./gradlew resolveConfigurations
-
-test:
-  override:
-    - ./gradlew check
-
-deployment:
-  # disable until we work through OJO publishing issues with Bintray
-  #develop:
-  #  branch: develop
-  #  owner: palantir/encrypted-config-value
-  #  commands:
-  #   - ./gradlew -i artifactoryPublish
-  release:
-    tag: /[0-9]+(\.[0-9]+)+/
-    owner: palantir
-    commands:
-      - ./gradlew -i bintrayUpload
+version: 2
+jobs:
+  build:
+    working_directory: /root/encrypted-config-value
+    environment:
+      ARTIFACTS_DIR: /tmp/artifacts
+      TESTS_DIR: /tmp/tests
+      PROJECTS: "encrypted-config-value encrypted-config-value-bundle encrypted-config-value-bundle-dropwizard1 encrypted-config-value-module"
```

Comment:
this is sorta scary, rest seems fine

can we instead teach gradle to do the copying?